src/c++/library/grpc_client.cc

```diff
@@ -367,6 +379,11 @@ InferResultGrpc::InferResultGrpc(
         std::make_pair(output.name(), std::make_pair(buf, byte_size)));
     index++;
   }
+  const auto& is_final_response_itr{
```

Comment:
Can this be extracted into a function so there isn't cut and paste in the two constructors?

Reply:
Much of these constructors is already copy and paste. Can we leave that kind of refactor for another time?